src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const stylesObj = {};
+  const lines = sourceString.split(';');
+
+  lines.forEach(function (line) {
+    const part = line.split(':');
+
+    if (part.length === 2) {
+      stylesObj[part[0].trim()] = part[1].trim();
+    }
+  });
```

Comment:
[CODE STYLE] - When using chained methods, ensure they have the same indentation to improve readability. If you switch to using map and filter, format them accordingly.